_fs/root-skel/etc/lighttpd.conf

```diff
@@ -62,7 +61,7 @@ server.pid-file = state_dir + "/lighttpd.pid"
 # we can still serve static files and output of the CGI scripts. The only
 # exception is the '/log' subdir, where we want to get 404 error, if we're trying
 # to download log file that does not exist.
-url.rewrite-repeat-if-not-file = ( "^/(?!log/)(.*)" => "/index.html" )
+url.rewrite-repeat-if-not-file = ( "^/(?!log/)" => "/index.html" )
```

Comment:
Is the `(.*)` part removed due to changes in the latest version of lighttpd?

This line should probably be removed from a generic configuration as it is specific for Single Page Applications. Maybe there should be a separate configuration file for such applications (for reference purposes) or left as a comment?

Reply:
This is a modification to remove excess, which can make a difference on embedded systems.  This is unrelated to any change in lighttpd.

Regex captures use more resources and can be abused in regex attacks (security).  In this case, there is no use of the regex capture, and therefore it can be safely removed without changing the intended semantics of the regex, which is a prefix match using a zero-width negative lookahead assertion (which is non-capturing).

Reply:
Ok, so it is redundant. That makes sense